src/figuresClasses.ts

```diff
@@ -1,11 +1,81 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: 'triangle' | 'circle' | 'rectangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = 'triangle';
+
+    const sumAB: number = this.a + this.b;
+    const sumAC: number = this.a + this.c;
+    const sumBC: number = this.b + this.c;
+
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('Error');
```

Comment:
[CODE STYLE] - When throwing an error, provide a detailed description in the error message to indicate what went wrong. For example, 'Triangle sides must be positive numbers.'